portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/upgrades/SAMLUpgrade.testcase

```diff
@@ -17,6 +17,7 @@ definition {
 
 	@description = "This is a use case for LRQA-64613. View SAML configuration after upgrade 7.3.10"
 	@priority = "5"
+	@uitest
```

Comment:
Also need to mark one of ViewSAMLWithEncryptionConfigurationAfterUpgradeXXXX test as uitest too.

Reply:
I agree